WooCommerce/Classes/ViewRelated/Reviews/ReviewsViewController.swift

```diff
@@ -499,10 +499,27 @@ private extension ReviewsViewController {
         }
 
         markAsReadCount += 1
-        let message = NSLocalizedString("All reviews marked as read", comment: "Mark all reviews as read notice")
-        let notice = Notice(title: message, feedbackType: .success)
+        let notice = Notice(title: Localization.Notice.allReviewsMarkedAsRead,
+                            feedbackType: .success)
```

Comment:
Nit: we can keep the call in one line if it doesn't exceed the character limit - to make it easier to read:
```suggestion
        let notice = Notice(title: Localization.Notice.allReviewsMarkedAsRead, feedbackType: .success)
```

Reply:
I thought that adding a new line makes it easier to read. Done in - https://github.com/woocommerce/woocommerce-ios/pull/5833/commits/d63de37cbea1b79fe0a140e6ccf3c780b3780cfe